chain/chain/src/runtime/mod.rs

```diff
@@ -182,7 +179,7 @@ impl NightshadeRuntime {
             last_validator_proposals,
             gas_limit,
             is_new_chunk,
-            is_first_block_with_chunk_of_version,
+            is_first_block_with_chunk_of_version: _,
```

Comment:
Delete `is_first_block_with_chunk_of_version` from `ApplyChunkShardContext`. Do not leave any new lines. Pull branch before proceeding.